### PR TITLE
update services to v2.0.0

### PR DESCRIPTION
**Pull request by stelios-ritual**

### CODE REVIEW

File: services/hf-client-inference/2.0.0/recipe.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
     },
     "inputs": [
         {
-            "id": "HF_TOKEN",
-            "path": "env.HF_TOKEN",
-            "description": "The Hugging Face API token.",
+            "id": "HF_INF_TOKEN",
```
Comment (evangelos-ritual) [inline services/hf-client-inference/2.0.0/recipe.json:19]: Why is the name changed? I think you can use only one type of token, right? That could be shared with TGI

**Review by evangelos-ritual — COMMENTED** [review]
Thinking if it would be a good idea to add the new 2.0.0 recipes separately instead of renaming the 1.0.0 ones. Wouldn't this be the right way of upgrading recipes in the future?

File: services/hf-client-inference/2.0.0/recipe.json (hunk 1 above)
Comment (stelios-ritual) [reply services/hf-client-inference/2.0.0/recipe.json:19]: two things

1. Although one _can_ use the same token, you can easily think of scenarios where you want different tokens with, say, different names, expiration dates, etc. Plus the prefix makes it super explicit which service you are setting it for, avoiding "accidental" token injection between services
2. per our discussion with arshan [here](https://github.com/ritual-net/infernet-monorepo-internal/pull/136), we agreed it's weird to mix pre-fixed with non-prefixed ENVs for services

**Comment by stelios-ritual** [comment]
> Thinking if it would be a good idea to add the new 2.0.0 recipes separately instead of renaming the 1.0.0 ones. Wouldn't this be the right way of upgrading recipes in the future?

I think it comes down to "do we want people running the 1.0.0 services, given the 2.0.0 are available"? I think the answer is no. People already running those services will not use the CLI to re-configure them. If they do, they should run 2.0.0

@evangelos-ritual

**Review by evangelos-ritual — APPROVED** [review]

**Pull request merged**